test/parallel/test-util-inspect.js

```diff
@@ -1213,3 +1213,129 @@ assert.doesNotThrow(() => util.inspect(process));
   assert.strictEqual(util.inspect(new NotStringClass()),
                      'NotStringClass {}');
 }
+
+{
```

Comment:
It looks like none of the examples here test objects with a custom `inspect` method. Can you add some coverage in for that?

Reply:
Done.